voucher/migrations/0004_auto_20190530_2227.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 2.1.7 on 2019-05-30 22:27
```

Comment:
This migration needs to be renamed for tests to pass.  `0004_voucher_course_data.py` or something like that.

Reply:
I was planning on squashing the migrations into one if that's ok.

Reply:
Fine with me